Update dependency on electron-packager to ^15.1.0 for Apple Silicon support

NielsLeenheer opened the pull request:
Apple Silicon is supported in electron-packager 15.1.0 or later.

NielsLeenheer committed:
Update dependency on electron-packaget to ^15.1.0 for Apple Silicon support

malept commented:
This appears to be blocking on updating the minimum Node version to 10 and updating the CI.

malept committed:
ci: use Node 10

malept committed:
fix!: require Node 10.12.0

BREAKING CHANGE: drops support for Node < 10.12.0

Electron Packager requires 10.12.0 (due to its dependencies).

malept commented:
Hm, CI isn't triggering.

malept commented:
Released as v12.0.0
